README.md

```diff
@@ -863,6 +863,9 @@ List of request URI mapped to the `jsp` servlet. Defaults to `['*.jsp', '*.jspx'
 #####`sessionconfig_sessiontimeout`
 Default session timeout for applications, in minutes. Defaults to `30`.
 
+#####`sessionconfig_trackingmode`
+Default session tracking mode for applications. Defaults to `URL`, `COOKIE` (unless the context parameter cookies=false) and `SSL` (if a connector used by this context has secure=true).
```

Comment:
Yes it looks clear to me, could you just reference some relevant documentation page?